cross/squidguardmanager/patches/squidguardmgr.cgi.patch

```diff
@@ -66,4 +82,108 @@
  our $IMG_DIR    = 'images';
  our $CSS_FILE   = 'squidguardmgr.css';
  our $JS_FILE    = 'squidguardmgr.js';
-
+@@ -48,7 +103,7 @@
+ our $SQUID_WRAPPER = '/var/www/squidguardmgr/squid_wrapper';
```

Comment:
it's the default variable in the file but it is overwriten by variable in the config file ;)